org.kde.lokalize.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "org.kde.lokalize",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "lokalize",
+    "rename-icon": "lokalize",
+    "finish-args": 
+    [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host"
+    ],
+    "separate-locales": false,
```

Comment:
I don't think we use this in any of the org.kde.XXX flathubs, any special reason for this one to be different?

Reply:
Still based on https://invent.kde.org/kukuhsyafaat/flatpak-kde-applications/-/blob/master/org.kde.lokalize.json#L11, try to remove and check it

Reply:
https://github.com/flathub/flathub/pull/1603/commits/0c2ab283f731a6ba5badc0935d188357c8f0630a